src/modules/articles.js

```diff
@@ -1,10 +1,16 @@
 import axios from "axios";
 
 const Articles = {
-  async index() {
-    let result = await axios.get("/articles");
-    return result.data.articles;
+  async index(category) {
+    let params = category && { category: category };
+    try {
+      let result = await axios.get("/articles/", { params: params });
```

Comment:
if you pass in the category param in the URL, you can actually stub out the different categories in cypress

```
let result
if (category) { 
  result = await axios.get(`/articles/?category=${category}`)
} else {
  result = await axios.get("/articles")
}
```